### DISCUSSION TITLE
Not supported on this API

### DISCUSSION THREAD

**Discussion by PierrunoYT** [open-webui/open-webui#1123] [General]
![grafik](https://github.com/open-webui/open-webui/assets/95778421/8da1f6d5-a9d7-4a6c-9251-dbc1ccd3c2c5)

**Comment by justinh-rahb** [comment]
Update your install, this error only presents for Claude 3 endpoints on old versions.

**Comment by PierrunoYT** [comment]
> Update your install, this error only presents for Claude 3 endpoints on old versions.

I'm using it with Pinokio an it says it's already up to date

**Comment by justinh-rahb** [comment]
> I'm using it with Pinokio an it says it's already up to date

And I am telling you that it is not. Because you wouldn't be getting that error if it was. Stop arguing, stop spamming, you won't fix your issue by doing so.

**Comment by PierrunoYT** [comment]
Okay sorry for posting it at multiple places :/
